docs/nightly/en/reference/sql/data-types.md

```diff
@@ -64,6 +64,126 @@ Output:
 | `TimestampMillisecond` | 64-bit timestamp values with milliseconds precision | 8 Bytes |
 | `TimestampMicroSecond` | 64-bit timestamp values with microseconds precision | 8 Bytes |
 | `TimestampNanosecond` | 64-bit timestamp values with nanoseconds precision | 8 Bytes |
+| `Interval`|  Time interval | 4 Bytes for `YearMonth`, 8 Bytes for `DayTime` and 16 Bytes for `MonthDayNano`|
+
+### Interval Type
+
+`Interval` type is used in scenarios where you need to track and manipulate time durations.It can be written using the following verbose syntax:
+
+```
+ quantity unit [quantity unit...]
+```
+
+* `quantity`:  is a number (possibly signed),
+* `unit`:   is `microsecond`, `millisecond`, `second`, `minute`, `hour`, `day`, `week`, `month`, `year`, `decade`, `century`, or abbreviations or plurals of these units; 
+
+The amounts of the different units are implicitly added with appropriate sign accounting. Unfortunately, GreptimeDB doesn't support writing the interval in the format of [ISO 8601 time intervals](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) such as `P3Y3M700DT133H17M36.789S` etc. But it's output support it.
```

Comment:
```suggestion
The amounts of the different units are implicitly added with appropriate sign accounting. Unfortunately, GreptimeDB doesn't support writing the interval in the format of [ISO 8601 time intervals](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) such as `P3Y3M700DT133H17M36.789S` etc. But it's output supports it.
```

> The amounts of the different units are implicitly added with appropriate sign accounting

Does this mean the final value of a compound interval equals to the sum of all units with signs applied to each unit respectively?

Reply:
> The amounts of the different units are implicitly added with appropriate sign accounting

I have the same question. What does this sentence mean?

Reply:
> Does this mean the final value of a compound interval equals to the sum of all units with signs applied to each unit > respectively?

Of, course